cebra/data/datasets.py

```diff
@@ -71,7 +72,7 @@ def __init__(self,
                  neural: Union[torch.Tensor, npt.NDArray],
                  continuous: Union[torch.Tensor, npt.NDArray] = None,
                  discrete: Union[torch.Tensor, npt.NDArray] = None,
-                 offset: int = 1,
+                 offset: Offset = Offset(1),
```

Comment:
```suggestion
                 offset: Offset = Offset(0, 1),
```

otherwise, a symmetric offset is chosen